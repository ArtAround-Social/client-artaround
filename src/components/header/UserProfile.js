import React from 'react';
import PropTypes from 'prop-types';
import userImage from './user2.jpg';


function UserProfile() {
  const name = 'Vasily Markov';
  return(
    <>
      {name}
      <img src={userImage} alt='user png' height='40'/>
    </>
  );
}

UserProfile.propTypes = {
  name: PropTypes.string.isRequired
};

export default UserProfile;
