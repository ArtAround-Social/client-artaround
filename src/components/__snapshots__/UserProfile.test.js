import React from 'react';
import { shallow } from 'enzyme';
import UserProfile from '../header/UserProfile';

describe('UserProfile component', () => {
  it('renders name', () => {
    const wrapper = shallow(
      <UserProfile name='Vasily Markov'/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
