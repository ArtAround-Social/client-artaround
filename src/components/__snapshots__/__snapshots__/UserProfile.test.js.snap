// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile component renders name 1`] = `
<Fragment>
  Vasily Markov
  <img
    alt="user png"
    height="40"
    src="/path/image"
  />
</Fragment>
`;
